docs/guide/compatibility.md

```diff
@@ -1,66 +1,155 @@
-| Enabling array arithmetic                          | =ARRAYFORMULA(A2:A5*B2:B5)                                                | By default, array arithmetic is disabled globally.<br><br>The ARRAYFORMULA function enables array arithmetic for the formula inside the function.<br><br>To enable array arithmetic globally, [set the `useArrayArithmetic` option to `true`](../api/interfaces/configparams.html#usearrayarithmetic). | By default, array arithmetic is disabled globally.<br><br>The ARRAYFORMULA function enables array arithmetic inside the function. | By default, array arithmetic is enabled globally.        |
-| Cell references inside inline arrays               | ={A1, A2}                                                                 | The array's value is calculated but not updated when the cells' values change.                                                                                                                                                                                                                         | The array's value is calculated and updated when the cells' values change.                                                        | ERROR: invalid array                                     |
-| SPLIT function                                     | =SPLIT("Lorem ipsum dolor", 0)                                            | This function works differently from Google Sheets version but should be sufficient to achieve the same functionality in most scenarios. Read SPLIT function description on [the Built-in Functions page](built-in-functions.html#text).                                                               | Different syntax and return value.                                                                                                | No such function.                                        |
-
-### Built-in function implementation differences
-
-Some built-in functions are implemented differently than in Google Sheets or Microsoft Excel.
+# Compatibility
+
+HyperFormula is designed to follow the [OpenDocument Standard](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html)
+while maintaining the compatibility with other popular spreadsheet software:
+- _Microsoft Excel_,
+- _Google Sheets_.
+
+In certain situations, it is not possible to be consistent with all three products due to:
+* inconsistencies between OpenDocument Standard, _Microsoft Excel_ and _Google Sheets_,
+* limitations of _Microsoft Excel_ or _Google Sheets_,
+* [technical limitations](technical-limitations.md) of HyperFormula in its current development stage.
+
+## Configuration
+
+#### Separator characters
+
+In other spreadsheet software, separators depend on the configured language and locale. You might want to set them up according to your internationalization settings.
```

Comment:
There will be a link to `Internationalization overview` page here once it's created.